src/stores/wagers.js

```diff
@@ -24,6 +24,34 @@ const newWagerModel = {
     })
 }
 
+function matchMarkets(originalMarkets, expectedMarkets) {
+    return expectedMarkets.map(expectedMarket => originalMarkets.find(market => market.key === expectedMarket.key));
+}
+
+function normalizeMarkets(section) {
+    const eventMarkets = section.events
+        .map(e => e.displayGroups.markets);
+
+    if(eventMarkets.length < 1) return section;
```

Comment:
this blows up if eventMarkets isn't defined either, maybe do a `!eventMarkets && <whatever>` here?

Reply:
I think it's always defined.  You have sections without an event array at all?